scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -153,15 +137,21 @@ task IsVcfOnChromosome {
 
     mkdir output
     touch output/~{output_vcf_name}
-    CHR=$(cat chrom.txt)
-    if [ $CHR = ~{chromosome} ]; then
-      cp ~{input_vcf} output/~{output_vcf_name}
-      echo "Including ~{input_vcf} as it is on chromosome ~{chromosome}."
-    else
-      touch output/~{output_vcf_name}
-      echo "Skipping ~{input_vcf} as it is not on chromosome ~{chromosome}."
-    fi
-  }
+    VCF_CHR=$(cat chrom.txt)
+    chromosomes=( ~{sep=' ' chromosomes} )
+
+    for i in "${cchromosomes[@]}"
+    do
+      if [ $VCF_CHR = $i ]; then
+        cp ~{input_vcf} output/~{output_vcf_name}
+        echo "Including ~{input_vcf} as it is on $i."
+        break
+      else
+        touch output/~{output_vcf_name}
+        echo "Skipping ~{input_vcf} as it is not on $i."
+      fi
+    done
+  >>>
 
   runtime {
     docker: "gcr.io/gcp-runtimes/ubuntu_16_0_4:latest"
```

Comment:
This version of Ubuntu EOL'd [more than 2 years ago](https://wiki.ubuntu.com/Releases)... 😬

Reply:
😬 indeed